src/states/elder/mod.rs

```diff
@@ -75,34 +72,30 @@ pub struct ElderDetails {
     pub peer_map: PeerMap,
     pub routing_msg_filter: RoutingMessageFilter,
     pub timer: Timer,
+    pub dev_params: DevParams,
 }
 
 pub struct Elder {
     network_service: NetworkService,
     full_id: FullId,
     is_first_node: bool,
-    /// The queue of routing messages addressed to us. These do not themselves need forwarding,
-    /// although they may wrap a message which needs forwarding.
+    // The queue of routing messages addressed to us. These do not themselves need forwarding,
+    // although they may wrap a message which needs forwarding.
     msg_queue: VecDeque<SignedRoutingMessage>,
     routing_msg_backlog: Vec<SignedRoutingMessage>,
     direct_msg_backlog: Vec<(P2pNode, DirectMessage)>,
+    relocate_backlog: VecDeque<RelocateDetails>,
```

Comment:
I'm not 100% sure this is the right place for this `relocate_backlog`.
1 - This is probably something we want as shared state between all Elders.
   => This needs to be at least in chain, but really better in shared_state.
   => i.e if we have some relocate backlog, and a new Elder, we want it to vote on it too.

2 - I'm thinking that most of the work for this should probably be in the chain.
  => Since we are triggering relocation when churn happen, might be good to always push it into backlog at the point we increase_age. Then we probably would want to do the vote in a function called from Approved::chain_poll (not sure how that fit, it could come out of `.chain_mut().poll()` i.e either AccumulatedEvent or something else.) I think that would be shared state. Would also leave us more option into when to do the relocation, do we want to process all online/offline consensused first?